packages/chain/contracts/Eth_broker.sol

```diff
@@ -217,7 +216,7 @@ contract Eth_broker {
         mutex()
         returns (bool)
     {
-        (uint256 daiNeeded, uint256 ethReceived) = _commonMint(
+        (uint256 daiNeeded, uint256 ethReceived, uint256 ethSpent) = _commonMint(
```

Comment:
The reason for the migration was due to shortcomings in the original environment (since Etherlime is no longer maintained). While I do agree that it is theoretically better to split into two PRs, this should have been done some time ago when the original migration was made. At this point my personal opinion would be that it's making extra labour for little gain.

If you are very definite about wanting to do this, I should be able to roughly simulate it by breaking out all of the work from the last commit and re-adding it back in, but I find that to be of very limited value.